samples/react-mega-menu/config/package-solution.json

```diff
@@ -3,15 +3,15 @@
   "solution": {
     "name": "react-mega-menu-client-side-solution",
     "id": "466eb049-ab57-4a5d-ae51-0a8221cdb984",
-    "version": "0.1.3",
+    "version": "0.1.4",
     "includeClientSideAssets": true,
     "isDomainIsolated": false,
     "features": [
       {
         "title": "Application Extension - Deployment of custom action.",
         "description": "Deploys a custom action with ClientSideComponentId association",
         "id": "0582bd9b-5570-4698-8ac8-a2aee2801bad",
-        "version": "0.1.0",
+        "version": "0.1.4",
         "assets": {
```

Comment:
we should use 4-digit versioning for SPFx like 0.1.4.0. 
Otherwise it will give an error when deploying to app catalog
![image](https://github.com/user-attachments/assets/659417fb-c7c8-4015-9d33-cf3db2598e2a)